source/life_expectancy_script.R

```diff
@@ -0,0 +1,142 @@
+library(tidyverse)
+library(readxl)
+
+file_path <- "data/life-expectancy/life-expectancy-17-19-tab3.xlsx"
+
+# Read in the sheets as a list --------------------------------------------
+
+sheets <-
+  file_path %>%
+  excel_sheets() %>%
+  purrr::set_names() %>%
+  purrr::map(read_excel, path = file_path)
+
+# Transforming the sheets --------------------------------------------------------
+
+sheets <- sheets[-c(1, 2)] # removing the metadata sheets
+
+transposed_sheets <- lapply(sheets, function(x) {

```

Comment:
This is an optional nit-pick: for naming I prefer suffixes over prefixes because that helps with autocomplete.